docs/api/1.x/_details-get-list.rst

```diff
@@ -1,6 +1,3 @@
-A ``Total-Objects`` response header indicates the total number of objects
-of the list (not the response, since it can be paginated).
```

Comment:
We should probably have a `_details-head-list.rst` where we mention that the header can be obtained with HEAD